### PR TITLE
Dim/fe2 fiddles

### PR DESCRIPTION
**Pull request by didimitrie**

### CODE REVIEW

File: packages/frontend-2/app.vue [hunk 1]
```diff
@@ -26,3 +26,13 @@ watchAuthQueryString()
 // Awaiting to block the app from continuing until mixpanel tracking is fully initialized
 await useMixpanelInitialization()
 </script>
+<style>
```
Comment (didimitrie) [inline packages/frontend-2/app.vue:29]: might be nuxt page transitions

File: packages/frontend-2/components/auth/OnboardingPanel.vue [hunk 2]
```diff
@@ -1,11 +1,11 @@
 <template>
```
Comment (didimitrie) [inline packages/frontend-2/components/auth/OnboardingPanel.vue:1]: ignore

File: packages/frontend-2/components/form/TextInput.vue [hunk 3]
```diff
@@ -36,7 +36,7 @@
         :type="type"
```
Comment (didimitrie) [inline packages/frontend-2/components/form/TextInput.vue:36]: ignore

File: packages/frontend-2/components/form/select/Users.vue [hunk 4]
```diff
@@ -29,6 +29,7 @@
             <UserAvatar
               v-for="user in value"
               :key="user.id"
+              :user="user"
               :avatar-url="user.avatar"
```
Comment (didimitrie) [inline packages/frontend-2/components/form/select/Users.vue:33]: okay to keep, needs removing avatar-url

File: packages/frontend-2/components/header/NavLink-old.vue [hunk 5]
```diff
@@ -0,0 +1,34 @@
+<template>
```
Comment (didimitrie) [inline packages/frontend-2/components/header/NavLink-old.vue:1]: ignore

File: packages/frontend-2/components/project/page/LatestItems.vue [hunk 6]
```diff
@@ -6,16 +6,18 @@
       <div
```
Comment (didimitrie) [inline packages/frontend-2/components/project/page/LatestItems.vue:6]: ignore

File: packages/frontend-2/components/project/page/latest-items/Models.vue [hunk 7]
```diff
@@ -1,5 +1,5 @@
 <template>
```
Comment (didimitrie) [inline packages/frontend-2/components/project/page/latest-items/Models.vue:1]: ignore

File: packages/frontend-2/components/project/page/latest-items/models/GridItem.vue [hunk 8]
```diff
@@ -1,28 +1,32 @@
 <template>
```
Comment (didimitrie) [inline packages/frontend-2/components/project/page/latest-items/models/GridItem.vue:1]: ignore

File: packages/frontend-2/components/project/page/models/ModelPreview.vue [hunk 9]
```diff
@@ -0,0 +1,26 @@
+<template>
```
Comment (didimitrie) [inline packages/frontend-2/components/project/page/models/ModelPreview.vue:1]: rename to "Preview"

File: packages/frontend-2/components/project/page/models/StructureItem.vue [hunk 10]
```diff
@@ -0,0 +1,219 @@
+<template>
```
Comment (didimitrie) [inline packages/frontend-2/components/project/page/models/StructureItem.vue:1]: delete/ignore

File: packages/frontend-2/components/project/page/models/StructureItem2.vue [hunk 11]
```diff
@@ -0,0 +1,231 @@
+<template>
```
Comment (didimitrie) [inline packages/frontend-2/components/project/page/models/StructureItem2.vue:1]: rename to structureditem normal

File: packages/frontend-2/lib/auth/composables/onboarding.ts [hunk 12]
```diff
@@ -53,7 +53,7 @@ export function useProcessOnboarding() {
       })
       .catch(convertThrowIntoFetchResult)
 
-    if (data?.activeUserMutations.finishOnboarding) {
+    if (data?.activeUserMutations.finishOnboarding && goToDashboard) {
```
Comment (didimitrie) [inline packages/frontend-2/lib/auth/composables/onboarding.ts:56]: flag 🛑

File: packages/frontend-2/lib/user/composables/avatar.ts [hunk 13]
```diff
@@ -19,16 +19,19 @@ export function useUserAvatarInternalsparams(params: {
     switch (size) {
```
Comment (didimitrie) [inline packages/frontend-2/lib/user/composables/avatar.ts:19]: ignorable/deletable

File: packages/frontend-2/pages/projects/[id].stories.ts [hunk 14]
```diff
@@ -1,230 +0,0 @@
-import { Meta, Story } from '@storybook/vue3'
```
Comment (didimitrie) [inline packages/frontend-2/pages/projects/[id].stories.ts:1]: do not delete!

File: packages/server/modules/core/loaders.ts [hunk 15]
```diff
@@ -138,7 +138,7 @@ export function buildRequestLoaders(ctx: AuthContext) {
         async (branchIds) => {
           const results = keyBy(
             await getBranchLatestCommits(branchIds.slice()),
-            'branchId'
+            'branchid'
```
Comment (didimitrie) [inline packages/server/modules/core/loaders.ts:141]: dubious, dim needs to investigate if this is actually fucked or not

File: packages/server/modules/core/repositories/branches.ts [hunk 16]
```diff
@@ -131,22 +134,59 @@ function getPaginatedProjectModelsBaseQuery<T>(
   return q
 }
 
+export async function getStructuredProjectModels(projectId: string) {
```
Comment (didimitrie) [inline packages/server/modules/core/repositories/branches.ts:137]: note edge case discussed with Fabs that makes this worth it: 
we have the following branches: `test/inner`, and `test/super` - branch `test` does not exist => we can't get away easily by just adding a `children` field to the branch model (unless we do a migration? - sounds complicated)

File: packages/frontend-2/app.vue (hunk 1 above)
Comment (didimitrie) [reply packages/frontend-2/app.vue:29]: yep

File: packages/frontend-2/components/auth/OnboardingPanel.vue (hunk 2 above)
Comment (didimitrie) [reply packages/frontend-2/components/auth/OnboardingPanel.vue:1]: note: since git exists, i've deleted these components
